devel/libecl/src/ecl_grid.c

```diff
@@ -2224,6 +2224,66 @@ static void ecl_grid_init_cell_nnc_info(ecl_grid_type * ecl_grid, int global_ind
     grid_cell->nnc_info = nnc_info_alloc(ecl_grid->lgr_nr);
 }
 
+/*
+  The function ecl_grid_add_self_nnc() will add a NNC connection
+  between two cells in the same index. Observe that there are two
+  peculiarities with this implementation:
+
+   1. In the ecl_grid structure the nnc information is distributed
+      among the cells. The main purpose of adding the nnc information
+      like this is to include the NNC information in the EGRID files
+      when writing to disk. Before being written to disk the NNC
+      information is serialized into vectors NNC1 and NNC2. It is the
+      ordering in the NNC1 and NNC2 vectors which must be correct, and
+      that is goverened by the nnc_index argument - i.e. the nnc_index
+      serves as an 'ID' for the NNC connections.
+
+      After all NCC information has been entered you should be certain
+      that all nnc_index values in the range [0,num_nnc] have been
+      applied.
+
+
+   2. To get valid NNC information to load in e.g. Resinsight the the
+      corresponding TRANNNC keyword must be added to the INIT file,
+      i.e. the calling scope must create a ecl_kw with
+      transmissibility values in parallell with adding NNC information
+      to the grid:
+
+         fortio_type * init_file = fortio_open_writer( "CASE.INIT" , ...
+         ecl_grid_type * grid ...
+         ecl_kw_type * trannnc_kw = ecl_kw_alloc( "TRANNC" , num_nnc , ECL_FLOAT_TYPE );
+
+         for (int i = 0; i < num_nnc; i++) {
+             int   g1 = ...
+             int   g2 = ..
+             float T  = ..
+
+             ecl_grid_add_self_nnc( grid , g1 , g2 , i );
+             ecl_kw_iset( trannc_kw , i , T );
+         }
+         ...
+         ecl_grid_fwrite_EGRID( grid , ... );
+         ecl_kw_fwrite( trannc_kw , init_file );
+
+*/
```

Comment:
this documentation string contains several missing n's.  Rewrite all `trannc*` to `trannnc*`